Christmas/christmas tree.scad

```diff
@@ -319,7 +332,11 @@ module top_trunk (height=66, thickness=2,slop=.4)
 
 module branch (height=80, slot=75, vertices=7, x=0, y=0, thickness=2,slop=.4)
 {
-	p = rands(vertices,vertices+4,1)[0];
+        p = vertices;
+        if (randomize == "Yes") {   // have no idea why random is not work
+            p = rands(vertices,vertices+4,1)[0];
+        }
+
```

Comment:
This part does not work if you select randomize = Yes. Please correct it if you know how.

Reply:
I got it in next release.